bitcoin/Cargo.toml

```diff
@@ -89,4 +89,4 @@ required-features = ["rand-std"]
 name = "sighash"
 
 [lints.rust]
-unexpected_cfgs = { level = "deny", check-cfg = ['cfg(bench)', 'cfg(fuzzing)', 'cfg(kani)', 'cfg(mutate)', 'cfg(rust_v_1_60)'] }
```

Comment:
OT but we should generate these in the build script so that this is automated.

Reply:
For as often as we change these it might not be worth the effort. But agreed in principle.

Reply:
I feel like every damn time we forget something. Automation prevents mistakes.